src/containers/notification/NotificationPanel.tsx

```diff
@@ -94,95 +97,102 @@ const NotificationPanel = ({
   }, [hasMore, status, fetchNotifications])
 
   return (
-    <AntModal
-      wrapClassName={styles.modalContainerWrapper}
-      className={styles.modalContainer}
-      visible={panelIsOpen}
-      onCancel={toggleNotificationPanel}
-      width={428}
-      getContainer={document.getElementById('navColumn') || document.body}
-      style={{ top: isElectron ? 95 : 37, left: 223, margin: 0 }}
-      closable={false}
-      footer={null}
-      mask={false}
-      destroyOnClose
+    <Popup
+      anchorRef={anchorRef}
+      className={styles.popup}
+      isVisible={panelIsOpen}
+      checkIfClickInside={(target: EventTarget) => {
+        if (target instanceof Element && anchorRef) {
+          return (
+            anchorRef.current.contains(target) ||
+            [...target.classList].some(c => c.includes('PopupMenu'))
```

Comment:
Is the `PopupMenu` a class hardcoded in the @audius/stems project?
Thoughts on something like `import navStyles from './NavHeader.module.css'` and then check against `navStyles.iconNotification` here instead?

Reply:
The `anchorRef` is used to check if the click is inside of the button that triggers the NotificationPanel. The second check is for the overflow menu on an individual notification itself. But I agree that using the classname is pretty hacky. 

I've been meaning to add `forwardRefs` to the stems components anyway and I think this is a perfect use case:
https://github.com/AudiusProject/audius-client/pull/496/commits/0d588281d739cbaf71aea483e2a455382e893cb6#diff-774e572244f6064b7880927b357385d2c46ace5b14dd2a0dba55c32c3e25c861R108

Any thoughts?